app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+from __future__ import annotations
+from typing import Union
+
+
+class Animal:
+
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        name = "{Name: " + str(self.name)
+        health = ", Health: " + str(self.health)
+        hidden = ", Hidden: " + str(self.hidden) + "}"
+        return name + health + hidden
```

Comment:
explicitly defining the string that should be returned will improve the readability
Just use f-strings and parentheses to split into lines